.github/workflows/ci-main.yml

```diff
@@ -11,6 +11,7 @@ env:
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     DOTNET_CLI_TELEMETRY_OPTOUT: true
     PROJECT_NAME: LocalStorage
+    TESTEXTENSIONS_PROJECT_NAME: $PROJECT_NAME.TestExtensions
```

Comment:
Thinking about it, it might be better to have two separate build pipelines, one for each package. That way they can be released independently, if required.

Reply:
That's a good idea. I can work on that.

Reply:
I reverted the change to this file and introduced a new `ci-testextensions.yml` pipeline definition (not loving the file name).